data/landSales/deadlines.ts

```diff
@@ -1,4 +1,5 @@
 const deadlines: {[sector: number]: number} =  {
+  0: Date.UTC(2100, 0, 1) / 1000, // Fri, 01 Jan 2100 00:00:00 GMT
```

Comment:
2100 ?

Reply:
Yes, so we avoid using / waiting for a real one for the tests